debian/populate_changelog.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Populate Changelog file with version and maintainer
+# too complex to do it in Makefile
+
+sed "s/0.0.0/$1/" debian/changelog.ex > debian/changelog
+DEBNAME=${2:-$(git config user.name)}
+DEBEMAIL=${3:-$(git config user.email)}
+echo $DEBNAME $DEBEMAIL
+echo " -- "${DEBNAME:-Maintainer not set}" <"${DEBEMAIL:-no_maintainer@nowhere}">  "$(date -R) >> debian/changelog
+
+cat debian/changelog
+
```

Comment:
`populate_changelog.sh` must be removed, dch must work

Reply:
Do we really have to use dch? It's too monstrous, another build dependency, and has limited capabilities of Maintainer setting

Reply:
Think we can have a script for one-time changelogs for now.

Reply:
changelog file creation moved to dch, script removed